Modules/FindCUDA.cmake

```diff
@@ -764,15 +796,17 @@ if(CUDA_USE_STATIC_CUDA_RUNTIME)
     else()
       unset(CMAKE_THREAD_PREFER_PTHREAD)
     endif()
-    if (NOT APPLE)
-      # Here is librt that has things such as, clock_gettime, shm_open, and shm_unlink.
+    if (CUDA_VERSION VERSION_LESS "7.0" AND NOT APPLE)
+      # Before CUDA 7.0, there was librt that has things such as, clock_gettime, shm_open, and shm_unlink.
       find_library(CUDA_rt_LIBRARY rt)
       if (NOT CUDA_rt_LIBRARY)
         message(WARNING "Expecting to find librt for libcudart_static, but didn't find it.")
       endif()
     endif()
-  endif()
-endif()
+  endif(UNIX AND NOT ANDROID)
+endif(NOT CUDA_USE_STATIC_CUDA_RUNTIME STREQUAL OFF)
+
+# MESSAGE("CUDA_TOOLKIT_INCLUDE: ${CUDA_TOOLKIT_INCLUDE} CUDA_VERSION_STRING: ${CUDA_VERSION_STRIN} CUDA_CUDART_LIBRARY: ${CUDA_CUDART_LIBRARY} CUDA_cudart_static_LIBRARY: ${CUDA_cudart_static_LIBRARY}")
```

Comment:
Yep, that's documentation - I modified original syntax together with adding the version check as it seemed to have changed. It;s a kinf of 'developer' doc though so I'd rather kill it.